src/main/java/net/fabricmc/loom/configuration/providers/minecraft/MinecraftMappedProvider.java

```diff
@@ -129,17 +132,17 @@ private void mapMinecraftJar() throws IOException {
 		}
 	}
 
-	public TinyRemapper getTinyRemapper(String fromM, String toM) throws IOException {
+	public static TinyRemapper getTinyRemapper(LoomGradleExtension extension, String fromM, String toM) throws IOException {
```

Comment:
Why the change to make this static and pass in the extension? You can get this instance from the extension and call it none staticailly

Reply:
Sadly not, this class is actually instantiated _after_ the access widener jar processor is created and setup. Since this method doesnt really use any instance state, i decided to make it (and the one for the compile classpath) static, but they should probably be moved.

Reply:
Ok, thats fine. Yes, I would move these out now 👍